packages/keyring-api/src/superstruct.test-d.ts

```diff
@@ -4,6 +4,7 @@ import { expectAssignable, expectNotAssignable } from 'tsd';
 
 import { exactOptional, object } from '.';
 
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
```

Comment:
Should we keep this unused vars?

Reply:
We need it to infer the type `ExactOptionalObject` and this file is mostly used for type testing, so we need to keep it.